File: Assignment12/program3.java
import java.util.Scanner;

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        int  i = 0;
        int j = 0;
        for(i = 1;i<=iRow;i++)
        {
           for(j= iCol;j>=1;j--)
           {
                 System.out.print(j+"\t");
           }

           System.out.println();

        }

    }

}

class program3
{
    public static void main(String[] args)
    { 
        int iVal1= 0;
        int iVal2 = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of Rows");
        iVal1 = sobj.nextInt();

        System.out.println("Enter the number of columns");
        iVal2 = sobj.nextInt();
        
        Pattern pobj = new Pattern();

        pobj.Display(iVal1, iVal2);
        
    }
}

File: Assignment16/program3.c
#include <stdio.h>

void Display(int iRow, int iCol)
{
    int i=0;
    int j=0;

    for(i=1;i<=iRow;i++)
    {
        for(j=1;j<=iCol;j++)
        {
            if(i==j)
            {
              printf("$\t");
            }
            else if(i>j)
            {
                printf("#\t");
            }
            else
            {
                printf("*\t");
            }

        }

        printf("\n");

    }


}

int main()
{
    int iVal1 = 0;
    int iVal2 = 0;

    printf("Enter the number of Rows:");
    scanf("%d",&iVal1);

    printf("Enter the number of Cols:");
    scanf("%d",&iVal2);

    Display(iVal1,iVal2);

    return 0;
}